tests/cli/test_main.py

```diff
@@ -44,9 +44,16 @@
 def config_and_cursor(tmp_path):
     with NamedTemporaryFile(mode='wt', dir=tmp_path) as configfile:
         with NamedTemporaryFile(mode='w+t', dir=tmp_path) as cursorfile:
-            configfile.write('cursor-file: {0}\n'.format(cursorfile.name))
-            configfile.flush()
-            yield (configfile, cursorfile)
+            def write_config(config):
```

Comment:
Want `config=None` here if that's the default case?

Reply:
Yep! Nice catch, fixed.